boards/x86/qemu_x86/qemu_x86_tss_pf.dts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2023 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <mem.h>
+
+#ifndef DT_DRAM_BASE
+#define DT_DRAM_BASE		0
+#endif
+#ifndef DT_DRAM_SIZE
+#define DT_DRAM_SIZE		DT_SIZE_K(4096)
+#endif
+
+#include <intel/lakemont.dtsi>
```

Comment:
Actually can I NAK this?  This is an abuse of devicetree.  How you service exceptions is an issue of SOFTWARE configuration.  It belongs in kconfig.  All i386+ CPUs have a TSS and can implement this.  You can build a TSS paging Zephyr image and run it on the same hardware that just booted a traditional image with USERSPACE=n.

It's fine to have a board variant to express the difference, but the choice shouldn't involve creating a new devicetree description for a theoretical piece of separate "hardware" the doesn't exist.